postownik/app/client/src/PostOnList.svelte

```diff
@@ -0,0 +1,101 @@
+<script>
+
+    export let title;
+    export let content;
+    export let type;
+    export let visible = true;
+</script>
+{#if visible}
+<div class="post {type}">
+    <h2>{title}</h2>
+    <p>{content}</p>
+    <div class="photo">
+    <img src="https://ocdn.eu/pulscms-transforms/1/J8dk9kpTURBXy80ZTRiYzliODhiNzJmOGEzNmM2ZTdjMmI2ZTAzMTJiNi5qcGeTlQPNAggnzQT8zQLPkwXNAxTNAbyTCaY2NWI3MjMGgaEwAQ/mariusz-pudzianowski.jpg" alt="">
+</div>
+</div>
+{:else}
+<div class="post invisible {type}">
+    <h2>{title}</h2>
+    <p>{content}</p>
+    <div class="photo">
+    <img src="https://ocdn.eu/pulscms-transforms/1/J8dk9kpTURBXy80ZTRiYzliODhiNzJmOGEzNmM2ZTdjMmI2ZTAzMTJiNi5qcGeTlQPNAggnzQT8zQLPkwXNAxTNAbyTCaY2NWI3MjMGgaEwAQ/mariusz-pudzianowski.jpg" alt="">
+</div>
+</div>
+{/if}
```

Comment:
Nie jestem pewien kiedy to miało by się przydać, ale może o czymś nie pomyślałem. Tak czy siak lepiej to rozwiązać po prostu nie wstawiając danego postu do listy w komponencie `List`. Data flow jest ważniejszy we frameworku niż CSS.

Reply:
A rozumiem, chciałeś tak ukryć post jeśli jest prywatny. No tak nie może być bo wtedy ktoś może sobie dosłownie usunąć klasę i post się pojawia. Generalnie tego typu rzeczy będą rozwiązane na backendzie. Ty tylko bierzesz posty które będą już przefiltrowane i je wszystkie wyświetlasz. Nie musisz przejmować się logiką typu czy użytkownik może coś wyświetlić czy nie.